recipes/prgenv-gnu/24.11/gh200/environments.yaml

```diff
@@ -0,0 +1,38 @@
+gcc-env:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+  mpi:
+      spec: cray-mpich@8.1.30
+      gpu: cuda
+  unify: true
+  specs:
+  - aws-ofi-nccl@master
+  - boost +chrono +filesystem +iostreams +mpi +python +regex +serialization +shared +system +timer
+  - cmake
+  - cuda@12.4
```

Comment:
Is there a specific reason for pinning the cuda version, at least this specifically (e.g. `@12` could also work?). Same for the osu/python versions below?

Reply:
Mainly so that when Spack is upgraded, or a new version of cuda is added to our site repo, the recipe concretises to the same versions (for some modicum of reproducibility)
No reason not to bump the versions if creating a new 24.11 stack.

Reply:
This was updated to 12.6.